src/main/java/com/exadel/discount/entity/Category.java

```diff
@@ -0,0 +1,34 @@
+package com.exadel.discount.entity;
+
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "categories")
+public class Category {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID",
+            strategy = "org.hibernate.id.UUIDGenerator"
+    )
+    private UUID id;
+
+    private String name;
+
+    @OneToMany(cascade = CascadeType.ALL,
+    fetch = FetchType.EAGER,
```

Comment:
Try to use as much lazy as possible. Don't use eager without real reason

Reply:
done